<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Console</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fontLabel">
        <property name="text">
         <string> Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFontComboBox" name="fontName">
          <property name="currentFont">
           <font>
            <family>DejaVu Sans</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fontSize">
          <property name="minimum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fontSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="textColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="textColorSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="backgroundColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="backgroundColorSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textColorLabel">
        <property name="text">
         <string>Text Color</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="backgroudColorLabel">
        <property name="text">
         <string>Backgroud Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="environmentBox">
     <property name="title">
      <string>Environment</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="path"/>
        </item>
        <item>
         <widget class="QPushButton" name="pathSelect">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSelect</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textColorSelect</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectTextColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundColorSelect</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectBackgroudColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathSelect</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>selectPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Settings</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Settings</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontName</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>Settings</receiver>
   <slot>changeFont(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Settings</receiver>
   <slot>changeFontSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Settings</receiver>
   <slot>clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFont()</slot>
  <slot>selectTextColor()</slot>
  <slot>selectBackgroudColor()</slot>
  <slot>selectPath()</slot>
  <slot>save()</slot>
  <slot>load()</slot>
  <slot>updateSettings()</slot>
  <slot>changeFont(QFont)</slot>
  <slot>changeFontSize(int)</slot>
  <slot>clicked(QAbstractButton*)</slot>
 </slots>
</ui>
